src/System.Private.CoreLib/Resources/Strings.resx

```diff
@@ -1933,6 +1933,9 @@
   <data name="BadImageFormat_ParameterSignatureMismatch" xml:space="preserve">
     <value>The parameters and the signature of the method don't match.</value>
   </data>
+  <data name="BadImageFormat_ResType_SerBlobMismatch" xml:space="preserve">
+    <value>The type serialized in the .resources file was not the same type that the .resources file said it contained. Expected '{0}' but read '{1}'.</value>
```

Comment:
Is .resources always correct here?  The error I was hitting was when using a .resx; was it compiling the .resx into a .resources behind the scenes, in which case this is technically correct but not what the developer necessarily sees?

Reply:
This was the text used in desktop verbatim.

Reply:
Ok.